client/src/pages/PostSolution.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, ChangeEvent, FormEvent } from "react";
+import { useParams } from "react-router-dom";
+import { Col, Container, Form, Row } from "react-bootstrap";
+import Button from "../components/Button.tsx";
+
+interface FormData {
+  title: string;
+  description: string;
+  diagram: File | null;
+}
+
+// TODO: Add form validation
+// TODO: Include data about the challenge like title
+
+const PostSolution: React.FC = () => {
```

Comment:
`:React.FC` was not included in any of the previous components. Why do we include it now?

Reply:
https://github.com/facebook/create-react-app/pull/8177